### PR TITLE
Small UI changes

### PR DESCRIPTION
**Pull request by chernandez7**
- added icons to buttons
- shows if cent are below or above note
- bit of css styling to buttons

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -52,15 +52,17 @@
...
   <!-- For shared styles, shared-styles.html import in elements.html -->
   <style is="custom-style" include="shared-styles"></style>
 
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+  <!-- Load in paper-buttons and av-icons -->
+  <link rel="import" href="bower_components/paper-button/paper-button.html">
+  <link rel="import" href="bower_components/iron-icons/av-icons.html">
 
+  <!-- Load Jquery -->
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
```
Comment (Rhernandez513) [inline app/index.html]: @chernandez7 

Is there a reason why your loading Javascript in the Head?  I usually load it under the footer (very last thing) to ensure no javascript events fire off before the entire page structure is loaded

File: app/index.html [hunk 2]
```diff
@@ -177,10 +178,10 @@ <h2 class="page-title">Saved</h2>
 
             <section data-route="tuning">
               <paper-material elevation="1" id='background'>
-                <h2 class="page-title">Tuning</h2>
                 <my-tuner></my-tuner>
-                <p><canvas id='tunerCanvas'
-                style='border: 1px solid rgb(170, 170, 170);left: 10px;top: 10px;width: 50%; -moz-user-select: none;'></canvas></p>
+                <p><canvas id='tunerCanvas' width="500" height="225" 
+                style='border: 1px solid rgb(170, 170, 170);left: 10px;top: 10px;width: 60%; float: right -moz-user-select: none;'></canvas>
```
Comment (Rhernandez513) [inline app/index.html]: Get the styling out of the structure, put it in the styles/ dir

you can use https://github.com/Rhernandez513/handsonhelping as a quick-and-dirty ref guide

File: app/index.html (hunk 1 above)
Comment (chernandez7) [inline app/index.html]: Ah, wasn't aware of that.

File: app/styles/app-theme.html [hunk 3]
```diff
@@ -43,6 +43,12 @@
     /* paper-menu */
     --paper-menu-background-color: #fff;
     --menu-link-color: #111111;
+
+    --paper-button-ink-color: var(--paper-pink-a200);
+    --paper-button-raised-keyboard-focus: {
+      background-color: var(--paper-pink-a200) !important;
+      color: white !important;
+    };
```
Comment (Rhernandez513) [inline app/styles/app-theme.html:51]: I like the use of the framework to style

File: app/elements/my-recorder/my-recorder.html [hunk 4]
```diff
@@ -3,34 +3,39 @@
...
-        ul { list-style: none; }
-        #recordingslist audio { display: block; margin-bottom: 10px; }
-    </style>
-    <p>
-    <input onclick='startRecording(this)' type='button' value='Start Recording'/>
-    <input onclick='stopRecording(this)' type='button' disabled value='Stop Recording'/>
-    </p>
-    <h2>Recordings</h2>
-     <ul id='recordingslist'></ul>
+  <style type='text/css'>
```
Comment (Rhernandez513) [inline app/elements/my-recorder/my-recorder.html]: Same thing here with the CSS separation

**Comment by chernandez7** [comment]
- added in #!
- Moved JS to end of body

**Comment by chernandez7** [comment]
- Moved styling out to app stylesheet

**Pull request merged**